packages/prosemirror-lwdita/src/commands.ts

```diff
@@ -181,7 +181,7 @@ export class InputContainer {
 /**
  * Render a dialog form for inserting the PR metadata
  */
-export function renderPrDialog(ghrepo: string, source: string, updatedXdita: string): void {
+export function renderPrDialog(ghrepo: string, source: string, branch: string, updatedXdita: string): void {
```

Comment:
tsdoc is missing params

Reply:
Fixed in b6962288f2f38f863a94da06d3e6f53b77fe42fa .